test/mri/test_time.rb

```diff
@@ -454,6 +454,13 @@ def test_strptime_empty
     assert_raise(ArgumentError) { Time.strptime('+09:00', '%z') } # [ruby-core:62351] by matz.
   end
 
+  def test_strptime_type_error
```

Comment:
it's a bit confusing but the MRI suite is owner and imported from Ruby itself.
thus these could get lost easily on the next import, preferably use Ruby Specs under *spec/ruby* (maybe there is one - it's just tagged to be skipped) or simply move this to *test/jruby/test_time.rb*

Reply:
Sure, will do. Thanks

Reply:
Done, moved to `test/jruby/test_time.rb`